### DISCUSSION TITLE
How to use dark theme with Coreui pro 4.1 ?

### DISCUSSION THREAD

**Discussion by cedricvoinnet** [coreui/coreui#317] [Q&A]
Hello !

I'm trying to use the dark theme with Coreui pro 4.1 but I think I'm doing something wrong because some of my colors are different of those of the [demo](https://coreui.io/demo/4.0/dark/colors.html).

Here is a dropdown-header with a bg-light : 
![image](https://user-images.githubusercontent.com/10754693/148797949-eea708c4-fe4a-4eee-86db-5ebe26b0d069.png)

Or here an icon and a badge with respectively a text-primary and a bg-primary :
![image](https://user-images.githubusercontent.com/10754693/148796670-defa2ddc-f4a1-475d-94af-6ab64a966ef8.png)

 It looks like it uses the bg-* and text-* colors from the light theme.

To setup the dark theme, I did as follow :

```scss
@import "coreui_override/variables"; // Here I override some default variables
@import "@coreui/coreui-pro/scss/coreui";
@import "@coreui/coreui-pro/scss/themes/dark";
```

Is there something I did wrong so that the colors used are those of the light theme?

**Comment by curry684** [comment]
I have the same issue, downgrading to 4.0.4 fixes most dark theme issues from what I saw.

Tried to get commercial support for this but the form at https://coreui.io/support/ has been broken for at least 24 hours now. Very disappointing to see you posted this issue here over a month ago and also no answer whatsoever.

**Comment by mrholek** [comment]
@cedricvoinnet if you want to use different bg on dark theme please add something like this `bg-light dark:bg-transparent`

bg-light - on default theme
dark:bg-transparent - transparent od dark theme
